e2e/util_test.go

```diff
@@ -149,15 +158,19 @@ func s5cmd(workdir, endpoint string) func(args ...string) icmd.Cmd {
 
 		cmd := icmd.Command(s5cmdPath, args...)
 		env := os.Environ()
-		env = append(
-			env,
-			[]string{
-				fmt.Sprintf("AWS_ACCESS_KEY_ID=%v", defaultAccessKeyID),
-				fmt.Sprintf("AWS_SECRET_ACCESS_KEY=%v", defaultSecretAccessKey),
-			}...,
-		)
+		if os.Getenv("AWS_ACCESS_KEY_ID") == "" || os.Getenv("AWS_SECRET_ACCESS_KEY") == "" {
```

Comment:
Why is this change necessary?

Reply:
I wanted to test the credentials case, to do that, I thought , it should use credentials setted by myself. Therefore, I added an if condition to check if any of the credentials are given. But the same test case worked same with `master` branch, therefore I deleted it.